pkg/kubelet/kubelet.go

```diff
@@ -1982,6 +1985,7 @@ func (kl *Kubelet) HandlePodAdditions(pods []*api.Pod) {
 }
 
 func (kl *Kubelet) HandlePodUpdates(pods []*api.Pod) {
+	defer func(sourcesReadyValue bool) { kl.sourcesReadyForUse = sourcesReadyValue }(kl.sourcesReady())
```

Comment:
I don't think this works correctly, as kl.sourcesReady() is evaluated a the binding to the deferred func, not at the end of the function.

Also, I think we want to continue to dynamically evaluate that function.  What is the value of statically storing it?